css-gaps-1/Overview.bs

```diff
@@ -0,0 +1,601 @@
+			ordered from start to end along the gap axis.
+			Let |break| be the used value of either 'column-rule-break' or 'row-rule-break',
+			whichever applies to the given gap.
+
+		<li>
+			If |endpoints| contains fewer than 2 items, return |pairs|.
+
+		<li>
+			Remove the first item from |endpoints|.
+			Let |subset| be a list that initially contains only the item removed from |endpoints|.
+
+		<li>
+			Remove the first item from |endpoints| and add it to the end of |subset|.
+
+		<li>
+			If |endpoints| is non-empty, and either of the following conditions is true...
+			<ol>
+				<li>
+					|break| is ''column-rule-break/spanning-item'',
+					and a line segment from the first item in |subset| to the first item in |endpoints|,
+					with the same width as the given gap,
+					does not intersect a child item in the container
+				<li>
+					|break| is ''column-rule-break/none''
+			</ol>
+			...then go back to step 4.
+
+		<li>
+			Let |start| be equal to the first item in |subset|.
+			Let |end| be equal to the last item in |subset|.
+
+		<li>
+			<a>Compute the offset</a> for |start| within the given gap.
+			Offset |start| by the result,
+			in the forward direction along the gap axis.
+		
+		<li>
+			<a>Compute the offset</a> for |end| within the given gap.
+			Offset |end| by the result,
+			in the reverse direction along the gap axis.
+
+		<li>
+			Add to |pairs| a tuple consisting of |start| and |end|.
+		
+		<li>
+			Go back to step 2.
+	</ol>
+
+	A <a>gap decoration</a> is painted between each pair of endpoints identified using the steps above.
+
+	<pre class='propdef shorthand'>
+		Name: gap-rule-break
+		Value: <<'column-rule-break'>>
+		Applies to: Same as 'column-rule-break' and 'row-rule-break'
+	</pre>
+
+	Sets the 'column-rule-break' and 'row-rule-break' properties to the same value.
+
+Adjusting gap decoration endpoints: The 'column-rule-outset', 'row-rule-outset', and 'gap-rule-outset' properties {#outset}
+---------------------------------------------------------------------------------------------------------------------------
+
+	<pre class='propdef'>
+		Name: column-rule-outset, row-rule-outset
+		Value: <<length-percentage>>
+		Initial: ''50%''
+		Applies to: <a>grid containers</a>, <a>flex containers</a>, <a>multicol containers</a>, and <a>masonry containers</a>
+		Inherited: no
+		Percentages: refer to the <a>crossing gap width</a>
+		Animation type: by computed value type
+	</pre>
+
+	These properties can be used to offset the endpoints of <a>gap decorations</a> relative to the
+	<a>gap intersection points</a> which would normally determine their endpoints.
+	The initial value of ''50%'' places the <a>gap decoration</a> endpoint in the center of the intersection.
+	With a value of ''0'', the <a>gap decoration</a> endpoint will coincide with the edge of the intersection.
+	Negative values may be used to recede from the intersection.
```

Comment:
Should this indicate direction? "...may be used to recede from the intersection in the negative direction", or is that implied with "recede"?

Reply:
Changed the wording a bit to clarify. In short: a negative value moves the endpoint away from the center of the intersection.